dist/gaia-header.js

```diff
@@ -2,9 +2,6 @@
 ;(function(define){define(function(require,exports,module){
```

Comment:
All changes to [gaia-component](http://github.com/gaia-components/gaia-component) must be landed there.